configen/utils.py

```diff
@@ -39,6 +39,10 @@ def type_str(t: Any) -> str:
                 name = str(t._name)
 
     args = get_args(t) if hasattr(t, "__args__") else None
+    if name == "Callable":
+        in_args_str = ", ".join([type_str(t) for t in args[0]])
+        out_args_str = args[1]
+        ret = f"{name}[{in_args_str}, {out_args_str}]"
```

Comment:
aren't `[in_args]` in sq brackets needed? e.g. `ret = f"{name}[[{in_args_str}], {out_args_str}]"`?

Reply:
see line 43